extensions/amp-list/0.1/amp-list.css

```diff
@@ -20,4 +20,91 @@ amp-list[load-more] [load-more-button].amp-visible,
 amp-list[load-more] [load-more-end].amp-visible {
   display: block;
   width: 100%;
-}
\ No newline at end of file
+}
+
+amp-list[load-more] [load-more-loading].i-amphtml-default-ui,
+amp-list[load-more] [load-more-failed].i-amphtml-default-ui,
+amp-list[load-more] [load-more-button].i-amphtml-default-ui {
+  height: 80px;
+  padding: 12px 0px;
+  box-sizing: border-box;
+}
+
+/* TODO: Standardize the system font string we use across the code base,
+         possibly by factoring it out into its own class. */
+amp-list[load-more] [load-more-loading].i-amphtml-default-ui,
+amp-list[load-more] [load-more-failed].i-amphtml-default-ui,
+amp-list[load-more] [load-more-button].i-amphtml-default-ui,
+.i-amphtml-amp-list-load-more-button {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+  font-weight: bold;
+  font-size: 14px;
+  text-transform: uppercase;
+  letter-spacing: .1em;
+
+  color: rgb(51,51,51);
+  text-align: center;
+}
+
+amp-list[load-more] [load-more-loading].i-amphtml-default-ui .i-amphtml-amp-list-load-more-spinner {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  margin: 8px 0px;
+
+  background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 40 40' xmlns='http://www.w3.org/2000/svg'%3E%3Cdefs%3E%3ClinearGradient id='grad'%3E%3Cstop stop-color='rgb(51,51,51)' stop-opacity='.75'%3E%3C/stop%3E%3Cstop offset='100%25' stop-color='rgb(51,51,51)' stop-opacity='0'%3E%3C/stop%3E%3C/linearGradient%3E%3C/defs%3E%3Cpath d='M11,4.4 A18,18, 0,1,0, 38,20' fill='none' stroke='url(%23grad)' stroke-width='1.725'%3E%3C/path%3E%3C/svg%3E");
+  animation: 1000ms amp-list-load-more-spinner linear infinite;
+}
+
+@keyframes amp-list-load-more-spinner {
```

Comment:
@sparhami did we have a more generic way / place to change this spinner or are we cool with merging this and refactoring later?